django/contrib/auth/forms.py

```diff
@@ -72,6 +72,7 @@ class UserCreationForm(forms.ModelForm):
         label=_("Password"),
         strip=False,
         widget=forms.PasswordInput,
+        help_text=password_validation.password_validators_help_text_html()
```

Comment:
add trailing comma so if more items are later added, we don't have to modify this line again.